src/index.js

```diff
@@ -149,9 +150,7 @@ server.get('/_health', async (req, res) => {
   }
 });
 server.get('/robots.txt', (req, res) => {
```

Comment:
I assume we'll remove this endpoint in a follow-up then? Right now I'm not seeing the uploaded file linked anywhere?

Reply:
No, this is such that crawlers can hit eg `customerimages.magnet.me` but also the Cloudfront bucket, and their `robots.txt` settings are equal